pandas/core/indexes/datetimes.py

```diff
@@ -2094,13 +2094,8 @@ def bdate_range(start=None, end=None, periods=None, freq='B', tz=None,
 def cdate_range(start=None, end=None, periods=None, freq='C', tz=None,
                 normalize=True, name=None, closed=None, **kwargs):
     """
-    **EXPERIMENTAL** Return a fixed frequency DatetimeIndex, with
-    CustomBusinessDay as the default frequency
-
-    .. warning:: EXPERIMENTAL
-
-        The CustomBusinessDay class is not officially supported and the API is
-        likely to change in future versions. Use this at your own risk.
```

Comment:
I also don't think we should remove this, unless we also remove the actual 'experimental' status of the CustomBusinessDay itself (which might be OK to do, I just don't know it)

Reply:
Okay, once we reach a consensus, I can either add this back or remove the additional warnings.

Reply:
yes this is ok
we have had this for quite a long time